ecmabot.js

```diff
@@ -18,6 +18,7 @@ var JSBot = function(profile) {
 	this.factoids = new FactoidServer(path.join(__dirname, "ecmabot-factoids.json"));
 	this.caniuse_server = new CanIUseServer;
 	this.executeRegex = /^((?:sm?|v8|js?|b|n|>>?|>>>>|\|)>)([^>].*)+/;
+	this.youtubeRegex  = /https?:\/\/www.youtube.com\/watch\?v=(\w+)/;
```

Comment:
I believe the best use for an "explicit trigger" would then be a command like `!yt` to make it go fetch and paste data after somebody else has already pasted a link. Then it would only occur if at least one user is interested.

youtu.be links won't be a problem, but, maybe the googleapi auth key will?